modules/govuk_scripts/manifests/init.pp

```diff
@@ -33,6 +33,14 @@
       provider => 'pip',
     }
 
+    # Verify the boto installation and all script dependencies are installed
+    exec { 'check_boto':
+          path    => ['/usr/bin', '/usr/sbin'],
+          command => ['/usr/bin/pip3 install boto3'],
+          require => Class['base::packages'],
```

Comment:
This `exec` does not have an `unless` and so will run on every puppet run. This shouldn't happen.